### DISCUSSION TITLE
Prerelease 0.0.2-alpha is out!

### DISCUSSION THREAD

**Discussion by pintariching** [pintariching/rustle#58] [Announcements]
I've just released the first version to play around with. You can install the CLI with cargo using `cargo install rustle_cli --version "0.0.2-alpha"` or play around with the `rustle` library!

**Comment by qwertydelle** [comment]
Now to expand parsing. Could you make a checklist of what currently needs to be implemented?

**Comment by erlend-sh** [comment]
Terrific work! 👏 

Once the checklist for further work is done, that’d be a great time to announce the pre-release on Reddit et.al., as per my [previous comment](https://github.com/pintariching/rustle/discussions/41).

**Reply by pintariching** [reply]
Okay I made a roadmap-ish list of some of Sveltes features. I looked at all the examples on the official site and derived what should be implemented from there. It's not much but it's a start!

**Comment by qwertydelle** [comment]
@pintariching this is a little bit of a nitpick but could we change the files in the example to `.svelte`?

**Reply by qwertydelle** [reply]
Feel like that is better since it's the convention for svelte files.

**Reply by pintariching** [reply]
Svelte files are going to be picked up by the svelte extension if you have it installed and it might parse files differently and return different results, or it might not I'm not sure on this.

Yeah sure we could try changing the extension, maybe even add different examples with a svelte extension just for show 😁
